README.md

```diff
@@ -131,12 +132,15 @@ export E2E_STAGING_PASSWORD=[e2e_password_value]
 The credentials should be for a cognito account which is 
 able to login without an MFA SMS. 
 
-Then (with appropriate AWS credentials) you can run
+Then (assuming you have the correct permissions) you can run
 
-`make e2e`
+```
+source behave_env.sh
+eval $(gds aws security-test -e); make e2e
```

Comment:
Should we have the gds cli instructions in a public repo?

Reply:
Er, I don't know. Gds-cli is marked as internal but not private and there are similar instructions in the same readme but happy to amend if need be. My reason for including it is that I was running it with the wrong aws user and it was sort of but not quite working - so it took me a while to figure out that it was the wrong user and this should prevent that.